components/brave_new_tab_ui/containers/newTab/settings/backgroundImage.tsx

```diff
@@ -39,64 +65,101 @@ class BackgroundImageSettings extends React.PureComponent<Props, {}> {
+              <SettingsText>{getLocale('showBackgroundImage')}</SettingsText>
+              <Toggle
+                onChange={toggleShowBackgroundImage}
+                checked={showBackgroundImage}
+                size='large'
+              />
+            </SettingsRow>
+            <SettingsRow isChildSetting={true}>
+              <SettingsText>{getLocale('brandedWallpaperOptIn')}</SettingsText>
+              <Toggle
+                onChange={toggleBrandedWallpaperOptIn}
+                // This option can only be enabled if
+                // users opt in for background images
+                checked={showBackgroundImage && brandedWallpaperOptIn}
+                disabled={!showBackgroundImage}
+                size='small'
+              />
+            </SettingsRow>
+            {showBackgroundImage && featureCustomBackgroundEnabled && (
+              <StyledCustomBackgroundSettings>
+                <StyledCustomBackgroundOption
+                  onClick={this.onClickCustomBackground}
+                >
+                  <StyledSelectionBorder selected={usingCustomBackground}>
+                    <StyledUploadIconContainer selected={usingCustomBackground}>
+                      <UploadIcon />
+                      <StyledUploadLabel>
+                        {getLocale('customBackgroundImageOptionUploadLabel')}
+                      </StyledUploadLabel>
+                    </StyledUploadIconContainer>
+                  </StyledSelectionBorder>
+                  <StyledCustomBackgroundOptionLabel>
+                    {getLocale('customBackgroundImageOptionTitle')}
+                  </StyledCustomBackgroundOptionLabel>
+                </StyledCustomBackgroundOption>
+                <StyledCustomBackgroundOption
+                  onClick={this.onClickBraveBackground}
+                >
+                  <StyledSelectionBorder selected={usingBraveBackground}>
+                    <StyledCustomBackgroundOptionImage style={{ backgroundImage: `url(${braveBackground})` }} selected={usingBraveBackground}/>
+                  </StyledSelectionBorder>
+                  <StyledCustomBackgroundOptionLabel>
+                    {getLocale('braveBackgroundImageOptionTitle')}
+                  </StyledCustomBackgroundOptionLabel>
+                </StyledCustomBackgroundOption>
+                <StyledCustomBackgroundOption
+                  onClick={this.onClickSolidColorBackground}
+                >
+                  <StyledSelectionBorder selected={usingSolidColorBackground}>
+                    <StyledCustomBackgroundOptionSolidColor
+                      style={{ backgroundColor: usingSolidColorBackground ? newTabData.backgroundWallpaper?.wallpaperSolidColor : '#151E9A' }}
+                      selected={usingSolidColorBackground}
+                    />
+                  </StyledSelectionBorder>
+                  <StyledCustomBackgroundOptionLabel>
+                    { getLocale('solidColorTitle') }
```

Comment:
Nit: Not sure whether we have any style recommendations on spaces around string interpolations but might be worth removing them to be consistent.